### PR TITLE
Camunda.Modeler version 5.7.0

### LINKED ISSUE #95877: [Package Request]: Camunda Modeler
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not a script file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: Camunda
Publisher Url: https://camunda.com
Package Name: Camunda Desktop Modeler
Description: BPMN Modeler
Package Version: 5.7.0
Installer URL: https://downloads.camunda.cloud/release/camunda-modeler/5.7.0/camunda-modeler-5.7.0-win-x64.zip

Current Package Identifier (for new versions): -

### PR DESCRIPTION
**Pull request by Miladiir**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/95988)

### CODE REVIEW

**Comment by Miladiir** [comment]
@microsoft-github-policy-service agree

**Comment by Trenly** [comment]
This is blocked by microsoft/winget-cli#2711; As a workaround, `start 'camunda modeler'` does seem to launch the program.

Start menu entries and desktop icons are not added for portable packages, which is intentional.

**Comment by Miladiir** [comment]
@Trenly thanks, that works. Do we want to merge this still, or wait for winget to support this use case first? I am willing to contribute for this package in the future.

**Comment by mdanish-kh** [comment]
> @Trenly thanks, that works. Do we want to merge this still, or wait for winget to support this use case first? I am willing to contribute for this package in the future.

If it does not launch as expected, merging it would mean a bad user experience.
cc @denelon - Maybe we can have something like a `.dll` label if we're blocking these submissions?

**Comment by Trenly** [comment]
There should be a Zip-Binary label already existing

**Comment by denelon** [comment]
I added:
"Depends on .dlls that aren't available via symlink" to the description of the "zip-binary" label. If it should be worded differently, just let me know. 😊

File: manifests/c/Camunda/Modeler/5.7.0/Camunda.Modeler.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Camunda.Modeler
+PackageVersion: 5.7.0
+PackageLocale: en-US
+Publisher: Camunda
+PackageName: Modeler
```
Comment (stephengillie) [inline manifests/c/Camunda/Modeler/5.7.0/Camunda.Modeler.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://camunda.com/
PackageName: Modeler
PackageUrl: https://camunda.com/platform/modeler/
```

**Comment by mdanish-kh** [comment]
> I added: "Depends on .dlls that aren't available via symlink" to the description of the "zip-binary" label. If it should be worded differently, just let me know. 😊

We would probably also need that label in winget-pkgs repo and the ability to tag issues and PRs with it through fabricbot 👀

**Comment by Trenly** [comment]
> > I added: "Depends on .dlls that aren't available via symlink" to the description of the "zip-binary" label. If it should be worded differently, just let me know. 😊
> 
> We would probably also need that label in winget-pkgs repo and the ability to tag issues and PRs with it through fabricbot 👀

Yes; I think the team is still working through FabricBot issues and are looking for it to be a bit more stable before additional updates

**Review by stephengillie — APPROVED** [review]

**Pull request merged**